stringResources/en-US/resources.resjson

```diff
@@ -62,8 +64,7 @@
 	"Visual_DisplayMonths": "Display months",
 	"Visual_DisplayWeeks": "Display weeks",
 	"Visual_DisplayDays": "Display days",
-	"Visual_Cursor": "Cursor",
-	"Visual_CursorColor": "Cursor color",
+	"Visual_PointerEdge": "Pointer edge",
```

Comment:
Seems we don't plan to use `Visual_PointerEdge` anymore?

Reply:
You're certainly right.